src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -9,7 +9,9 @@ public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays a summary of all commands available in the system.\n"
+            + "Parameters: None"
```

Comment:
Add in a newline `\n` after "Parameters: None"

Reply:
Thanks for spotting